doc/specifications/rest/REST-API-V2.rst

```diff
@@ -1454,6 +1455,24 @@ Delete Content Version
     :401: If the user is not authorized to delete this version
     :403: If the version is in state published
 
+Delete Content Version Translation
+``````````````````````
+:Resource: /content/objects/<ID>/versions/<versionNo>/translations/<language_code>
+:Method: DELETE
+:Description: The translation is removed from a draft
```

Comment:
according to https://ezsystems.app.box.com/s/d4jsidn3m0zzj39m9aei24357ajaeqfu this needs to be possible on the published version as well

Reply:
> according to https://ezsystems.app.box.com/s/d4jsidn3m0zzj39m9aei24357ajaeqfu this needs to be possible on the published version as well

You're right, it's just a `remove` link in the spec, without any intermediate draft view.

I'll change it, so the REST endpoint will:
1. Check if Version is published (as box spec shows this only for published Versions).
2. Create Draft.
3. Remove language from this Draft.
4. Publish Draft w/o removed language.

@dpobel does this work for you?